vme/src/descriptor_data.cpp

```diff
@@ -310,7 +310,7 @@ char *descriptor_data::getCommandHistory()
 
 void descriptor_data::setCommandHistory(const char *value)
 {
-    strncpy(history, value, sizeof(history));
+    strncpy(history, value, sizeof(history) - 1);
```

Comment:
Warning here that strncpy will not null terminate if size matches - so do 1 less

Reply:
Maybe also null this last char